code/datums/components/turf_decals/td_paint.dm

```diff
@@ -0,0 +1,47 @@
+/datum/component/turf_decal/stripes/line
```

Comment:
TYPE PATHS ARE NOT A DATABASE REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

Reply:
It's a necesary evil to allow just that decal's overlay to rotate.

Reply:
Unless you meant the icon information should just be given in through initialize, that I can do easily.

Reply:
yeah instead of using types here just pass the icon state to `/datum/component/turf_decal/Initialize()`